CHANGELOG.md

```diff
@@ -1,6 +1,30 @@
 Magento Functional Testing Framework Changelog
 ================================================
 
+2.4.0
+-----
+### Enhancements
+* Maintainability
+    * Added new `mftf static-checks` command to run existing static checks against the attached test codebase
+        * See DevDocs for details
+    * Added new `mftf generate:docs` command that generates documentation about attached test codebase
+        * See DevDocs for details
+* Traceability
+    * Allure reports for tests now contain collapsible sections for `actionGroup`s used in execution.
+
+### Fixes
+* Fixed an issue where `magentoCli` would treat `argument="0"` as a null value.
+* Fixed an issue where `amOnPage` and `waitForPwaElementVisible` would not utilize the `timeout` attribute correctly.
+* Fixed an issue where invalid XML characters would cause Allure to throw an exception without a resulting report.
+* Fixed `codeception.dist.yml` configuration for keeping previous test run results.
+* PHP Notices are no longer thrown then XML is missing non-necessary attributes.
+* Removed unusable `fillSecretField` action from schema.
```

Comment:
Action group context (MQE-1472) is not mentioned. Is this on purpose?

Reply:
MQE-1472 was addressed in line 13

Reply:
Ah, I see